content/rancher-prime/azure/contents.lr

```diff
@@ -0,0 +1,282 @@
+title: Rancher Prime in Azure: Usage Instructions
+---
+theme: rancher
+---
+body:
+
+## Prerequisites <a name="prerequisites"></a>
+
+1. A Rancher-compatible AKS cluster. Please see
+   [Rancher support matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-7-6/)
```

Comment:
is there an unversioned version of this URL or will we need to update the url every time Rancher updates?

Reply:
Good question. That's the latest support matrix that is available on the Rancher doc site so we'll go with this one.